src/stats/actor.rs

```diff
@@ -47,6 +49,23 @@ impl StatsActor {
             }
         });
 
+        let hub_stats_task = tokio::spawn({
+            let hub = self.hub.clone();
+            let update_period = Arc::clone(&self.update_period);
```

Comment:
is there a reason for not calling clone directly ?

Reply:
there is no difference, that's a more _expressive_ way to saying you are cloning the ARC, not the data, but we can use .clone, no worries!

Reply:
... or just let's use it :laughing: 

![image](https://github.com/user-attachments/assets/684e7ac8-1045-4f69-8cfc-e514087fe607)